daemon/mgr/container_utils_test.go

```diff
@@ -206,7 +206,7 @@ func Test_mergeEnvSlice(t *testing.T) {
 				newEnv: []string{"a=b"},
 				oldEnv: []string{"c=d"},
 			},
-			want:    []string{"c=d", "a=b"},
```

Comment:
I do not advise this way to fix, try to match every value in slice. But consider you have sort before, it also work.

Reply:
Agree. we should sort the slice before we check.